### DISCUSSION TITLE
如何找到中文文档的源代码

### DISCUSSION THREAD

**Discussion by Yueyanc** [element-plus/element-plus#4680] [Q&A] [answered]
我在element-plus源码中没有发现支持中文，但是官方网站是支持的，请问我该去哪里找到中文支持源码

**Comment by Yueyanc** [comment, marked as answer]
我已经找到了，链接如下：https://crowdin.com/project/element-plus/zh-CN

**Reply by yaolunmao** [reply]
强啊 你是怎么发现的啊

**Reply by zlecheng** [reply]
> 我已经找到了，链接如下：https://crowdin.com/project/element-plus/zh-CN

这玩意咋下载的哦

**Comment by Dreamhai** [comment]
下载不了哇，怎么还要权限，要加入？

**Comment by rzzf** [comment]
See https://github.com/element-plus/element-plus/discussions/24694
